patches/chrome-android-java-src-org-chromium-chrome-browser-toolbar-top-ToolbarPhone.java.patch

```diff
@@ -11,7 +11,19 @@ index fd3724b04d07cc2dedeff2a1f629175b5b035446..74a09ee1886837e90a7bf5bbfe8ae5db
                                                             NewTabPage.OnSearchBoxScrollListener,
                                                             TabCountObserver {
      /** The amount of time transitioning from one theme color to another should take in ms. */
-@@ -425,10 +425,12 @@ public class ToolbarPhone extends ToolbarLayout implements Invalidator.Client, O
+@@ -377,9 +377,11 @@ public class ToolbarPhone extends ToolbarLayout implements Invalidator.Client, O
+ 
+             setLayoutTransition(null);
+ 
++/*
+             if (getMenuButtonWrapper() != null) {
+                 getMenuButtonWrapper().setVisibility(View.VISIBLE);
+             }
++*/
```

Comment:
it is possible to make it without patches, if I do `getMenuButtonWrapper().setVisibility(View.GONE);` in BraveToolbarLayout after that call, but it looks ugly as you see the 3 dots menu for a bit and after that it's gone.

Reply:
can we change getMenuButtonWrapper to return null?

Reply:
or do we need it sometimes?

Reply:
@bridiver that's was the reason of the crash. We had it to make null in that method https://github.com/brave/brave-core/pull/4690/files#diff-a6c2c460ae34706b2d235a605f16caa3L176 but it crashed in chromium sources on that place https://github.com/brave/browser-android-tabs/blob/base-80.0.3987.117/chrome/android/java/src/org/chromium/chrome/browser/modaldialog/TabModalPresenter.java#L330 because that is https://github.com/brave/browser-android-tabs/blob/base-80.0.3987.117/chrome/android/java/src/org/chromium/chrome/browser/modaldialog/TabModalPresenter.java#L297 null

Reply:
we don't need it, chromium uses it in some places still, or they just forgot to add a null check there. Alternately we can add a check to TabModalPresenter, it works that way.

Reply:
resolved via DMs